src/helper-methods.js

```diff
@@ -0,0 +1,67 @@
+import merge from "lodash/object/merge";
+import { Chart, Style } from "victory-util";
+import { Transform } from "react-art";
+
+
+module.exports = {
+  getStyles(props, defaultStyles) {
+    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
```

Comment:
Starting in lodash 4 all operations are immutable (:tada:) - probably worth ticketing out the upgrade somewhere and removing all the `merge/extend({}, ...)` stuff.